tests/test_style.py

```diff
@@ -33,19 +33,19 @@ def test_colormapstyle_process(self):
         expected = pd.DataFrame(
```

Comment:
Why is this style test change needed?

Reply:
As I said in PR description:

I suspect, that new version ``seaborn`` was released and this changed this values. This PR should not change anything related with ``test_style`` test and this test is also failing on current master (with newest version of ``seaborn``)